app/javascript/packages/document-capture/components/documents-step.tsx

```diff
@@ -32,9 +32,11 @@ export function DocumentCaptureSubheaderOne({
 export function SelfieCaptureWithHeader({
   defaultSideProps,
   selfieValue,
+  selfieCaptureEnabled,
 }: {
   defaultSideProps: DefaultSideProps;
   selfieValue: ImageValue;
+  selfieCaptureEnabled: boolean;
```

Comment:
Initially I thought maybe it'd be better / simpler to add `selfieCaptureEnabled` to the `DefaultSideProps`, since that's already being passed around where needed. But `DefaultSideProps` is a set type, and I couldn't figure out how to change it. Perhaps someone with more Typescript knowledge will know of a way to do that though.

Reply:
@night-jellyfish , may be

```typescript
type DefaultSideProps = Pick<
  FormStepComponentProps<DocumentsStepValue>,
  'registerField' | 'onChange' | 'errors' | 'onError'
> | { selfieCaptureEnabled: boolean }; 
```